docs/extensions/civix.md

```diff
@@ -1 +1,582 @@
-Coming soon!
+# Civix
+
+>> ![information](../img/info.png)
```

Comment:
I like these *information* and *danger* blocks. Great idea! 

As an aside: I wonder if (in the future) we might be able to find a way to make this easier. I'm imagining something like 

```
{information
I am an informational message.
}
```

(Not a project that I feel should be investigated before merging this PR, but just a random thought.)